internal/pkg/deploy/cloudformation/workload.go

```diff
@@ -39,7 +39,8 @@ func (cf CloudFormation) pushWorkloadTemplateToS3Bucket(bucket string, config St
 		return "", fmt.Errorf("generate template: %w", err)
 	}
 	reader := strings.NewReader(template)
-	url, err := cf.s3Client.Upload(bucket, fmt.Sprintf(fmtWorkloadCFNTemplateName, config.StackName(), sha256.Sum256([]byte(template))), reader)
+	artifactName := path.Join(config.StackName(), fmt.Sprintf("%x.yml", sha256.Sum256([]byte(template))))
+	url, err := cf.s3Client.Upload(bucket, s3.TemplateArtifactPath(artifactName), reader)
```

Comment:
What do you think of a path like:
```
templates/<stack name>/<sha>.yml
```

Reply:
Currently it's `manual/templates/stackname/sha.yml` what do ya think? The new code here doesn't change the path it's just some refactoring!